### DISCUSSION TITLE
Portal function?

### DISCUSSION THREAD

**Discussion by kartogishenko** [hajkmap/Hajk#1209] [Q&A] [answered]
Hi, newbie question as we are trying out the platform as of today: are there any portal function if you have more than 1 map?

**Comment by Hallbergs** [comment, marked as answer]
Hello, there is a possibility to enable a control-button which you can use to toggle between your maps. The setting used to enable this button ("Visa kartväljare") is located under the map-settings, see image below: 
![image](https://user-images.githubusercontent.com/43059093/194801293-f4bef0f5-0306-4688-8975-0634748a6ebb.png)

When enabled, a button to switch between the available maps will appear:
![image](https://user-images.githubusercontent.com/43059093/194826842-4c293907-5ecb-406b-bb77-7a6e0488a87a.png)
